### PR TITLE
Updates for Katello 3.13.0.

### PR DESCRIPTION
**Pull request by jjeffers**

### CODE REVIEW

**Review by evgeni — APPROVED** [review]

**Review by jturel — CHANGES_REQUESTED** [review]
The 'latest' metadata should be updated across all the katello versions

File: plugins/katello/3.12/installation/index.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: plugins/katello/documentation
 title: Katello Installation
 version: '3.12'
 foreman_version: 1.22
-latest: '3.12'
+latest: '3.13:'
```
Comment (jturel) [inline plugins/katello/3.12/installation/index.md]: extra ':' snuck in

Comment (jjeffers) [reply plugins/katello/3.12/installation/index.md]: :(

**Review by jturel — APPROVED** [review]
LGTM!

**Review by evgeni — APPROVED** [review]

**Pull request merged**